keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1238,12 +1502,16 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+TBD
+
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
 <!--
 Even if applying deprecation policies, they may still surprise some users.
 -->
 
+No.
+
 ### Monitoring Requirements
 
 <!--
```

Comment:
What metrics are we going to use to ensure this feature is (1) enabled, (2) how much is it enabled (i.e. how many sidecars are currently in use, and (3) if the functionality is broken in some capacity?

Reply:
For the question:

> ###### How can an operator determine if the feature is in use by workloads?

We should use the `kubernetes_feature_enabled` metric. See https://kubernetes.io/docs/reference/instrumentation/metrics/ for more details

Reply:
Also the below questions should all be answered.

Reply:
> For the question:
> 
> > ###### How can an operator determine if the feature is in use by workloads?
> 
> We should use the `kubernetes_feature_enabled` metric. See https://kubernetes.io/docs/reference/instrumentation/metrics/ for more details

I think we were planning on having such metric @gjkim42

Reply:
You also need to answer the following questions in this section, i.e.:

> How can an operator determine if the feature is in use by workloads?
> How can someone using this feature know that it is working for their instance?
> What are the reasonable SLOs (Service Level Objectives) for the enhancement?

etc.

Reply:
I've added some answers, for the rest I need to discuss with the working group today.

Reply:
> > For the question:
> > > ###### How can an operator determine if the feature is in use by workloads?
> > 
> > 
> > We should use the `kubernetes_feature_enabled` metric. See https://kubernetes.io/docs/reference/instrumentation/metrics/ for more details
> 
> I think we were planning on having such metric @gjkim42

I think we already have this metric. We don't have to implement our own, right?

Reply:
Indeed:
```
$ kubectl get --raw /metrics  | grep SidecarContainers
kubernetes_feature_enabled{name="SidecarContainers",stage="ALPHA"} 0
```